src/azure-cli/requirements.py2.Darwin.txt

```diff
@@ -26,7 +26,7 @@ azure-mgmt-billing==0.2.0
 azure-mgmt-botservice==0.2.0
 azure-mgmt-cdn==3.1.0
 azure-mgmt-cognitiveservices==5.0.0
-azure-mgmt-compute==6.0.0
+azure-mgmt-compute==7.0.0
```

Comment:
Can you declare package upgrade in changelog?

Reply:
> 
> 
> Can you declare package upgrade in changelog?

OK